libjulia/interpreter/EVMInstructionInterpreter.cpp

```diff
@@ -0,0 +1,393 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * IULIA interpreter module that evaluates EVM instructions.
+ */
+
+#include <libjulia/interpreter/EVMInstructionInterpreter.h>
+
+#include <libjulia/interpreter/Interpreter.h>
+
+#include <libsolidity/inlineasm/AsmData.h>
+
+#include <libevmasm/Instruction.h>
+
+#include <libdevcore/SHA3.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::julia;
+
+namespace
+{
+
+template <class S> S divWorkaround(S const& _a, S const& _b)
```

Comment:
We have a couple of copies this now - shouldn't it be moved to `devcore`?

Reply:
yep, good idea